huntsman/observatory.py

```diff
@@ -472,43 +355,217 @@ def _create_autoguider(self):
 
         self.autoguider = guider
 
-    def _create_flat_field_observation(self, alt=None, az=None,
-                                       dither_pattern_offset=5 * u.arcmin,
-                                       dither_random_offset=0.5 * u.arcmin,
-                                       n_positions=9
-                                       ):
+    def _create_flat_field_observation(self, alt=None, az=None, *args, **kwargs):
+
+        # Specify coordinates for flat field
         if alt is None and az is None:
             flat_config = self.config['flat_field']['twilight']
             alt = flat_config['alt']
             az = flat_config['az']
+            self.logger.debug(f'Using flat-field alt/az from config: {alt}, {az}.')
```

Comment:
It will always be helpful to see what alt/az is going in, not only when config values are used.
```suggestion
        self.logger.debug(f'Making flat-field observation for alt/az: {alt:.03}, {az:.03f}.')
```

Reply:
Really the `altaz_to_radec` should have a similar logging call.